rock.osdeps

```diff
@@ -34,7 +34,10 @@ binding_of_caller:
     gem: binding_of_caller
 
 activesupport:
-    gem: activesupport <= 4.2.5.1
+  ubuntu: ruby-activesupport
+  debian: ruby-activesupport
+  default:
+    gem: activesupport
```

Comment:
the current activesupport 5.0 is only compatible to ruby >= 2.2 

Installing the osdep is just the cleaner fix for the issue rather than adding a version contraint.
https://github.com/rock-core/package_set/pull/70


The osdep should always fit the ruby version installed on the system.

For the gem version you can't tell whether someone uses an older ruby, or has the current version installed and wants to install the latest version of activesupport. Not having the version dependency is more future proof than fixing the version. 

Also because one tends to forget why the version constraint was introduced.